pkgs/development/libraries/gpgme/default.nix

```diff
@@ -9,11 +9,13 @@ let
   gpgProgram = if useGnupg1 then "gpg" else "gpg2";
 in
 stdenv.mkDerivation rec {
-  name = "gpgme-1.8.0";
+  name = "gpgme-1.7.1";
```

Comment:
No, not in general, although 1.8.0 was particularly hostile to MacOS :-) — they meant to do [this](https://git.gnupg.org/cgi-bin/gitweb.cgi?p=gpgme.git;a=blobdiff;f=lang/cpp/src/GpgmeppConfig.cmake.in.in;h=cbe97139740c37e663d14fdb98d2cd61a8688053;hp=928d19f8ab155fe36a4d9f0e76115ae6e9213c04;hb=572c1aac107125ce62230251713349348373db5a;hpb=af79327021b5b3a5c4accd5d4e9f0124199bfdcc), which is a patch we might as well add rather than splitting gpgme_1_7 out of gpgme, and see [nixpkgs workaround](https://github.com/NixOS/nixpkgs/commit/d1a9f2e8be9624e06d42714a107d2b8288945d12) for another regression in 1.8.0. Both of this were fixed in gpgme master, and I believe we should better wait for 1.8.1 or 1.9.0 on 1.7.1.